### ISSUE TITLE
Is there a way to disable the creation of alt sound logs?

### ISSUE THREAD

**Issue by CoffeNook** [vpinball/vpinball#1701]
Every time I play a table, VPX creates a log file called 'altsound.' Is there a way to disable this feature?

**Comment by toxieainc** [comment]
@droscoe should know

**Comment by droscoe** [comment]
> Every time I play a table, VPX creates a log file called 'altsound.' Is there a way to disable this feature?

No, there is currently no way to disable.  The default configuration for altsound produces a minimal file, and it's overwritten every time VPX is shutdown, so it does not accumulate.  Is there a particular reason you would want it disabled?

**Comment by Heartman30044** [comment]
Let me just add my reason for having someway to disable it.  I have my drive organized where each table is in its own subdirectory.  Therefore, I end up with this file being duplicated with every table.  I understand that if the tables were in one large directory there would only be one logfile.

It's not a real big deal, because I can have the "kill" script in the front end, delete it.

**Comment by logoliv** [comment]
> It only creates the file when altsound is used

Since I installed PinMAME v3.6 the altsound.log file is created everytime I launch a table, although my altsound mode is set to zero for all my tables and I never used it...
